tests/integration/main.py

```diff
@@ -40,25 +40,22 @@ def generate_configuration():
             key_secret=envs['SECRET']
         )
     configuration = Configuration(**params)
+    configuration.debug = False
     configuration.apply_logging_config()
+
     return configuration
 
 
 def main():
-    args = sys.argv[1:]
     configuration = generate_configuration()
     api_client = ApiClient(configuration)
     workflow_executor = WorkflowExecutor(configuration)
 
-    if len(args) == 1 and args[0] == '--orkes-clients-only':
-        TestOrkesClients(configuration=configuration).run()
-    elif len(args) == 1 and args[0] == '--workflow-execution-only':
-        run_workflow_execution_tests(configuration, workflow_executor)
-    else:
-        test_async.test_async_method(api_client)
-        run_workflow_definition_tests(workflow_executor)
-        run_workflow_execution_tests(configuration, workflow_executor)
-        TestOrkesClients(configuration=configuration).run()
+    test_async.test_async_method(api_client)
```

Comment:
I had added this for convenience. This was removed by mistake or intentionally?

Reply:
removed since we can just run all the tests.